### PR TITLE
add npm autoupdate config on fingerprintjs

### LINKED ISSUE #7721: [Request] Add auto-update config on fingerprintjs
**Library name:** fingerprintjs
**Git repository url:** https://github.com/Valve/fingerprintjs

https://cdnjs.com/libraries/fingerprintjs

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7721 
@kennynaoh Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Valve/fingerprintjs
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/fingerprintjs
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1784
   - Count of watchers: 134
   - Count of forks: 279
 * NPM download stats (optional):
   - Downloads in the last day: 77
   - Downloads in the last week: 313
   - Downloads in the last month: 1434

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7721, cc @Valve

### CODE REVIEW

File: ajax/libs/fingerprintjs/package.json [hunk 1]
```diff
@@ -11,5 +11,14 @@
...
     "url": "https://github.com/Valve/fingerprintjs"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "npmName": "fingerprintjs",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "fingerprintjs*"
```
Comment (PeterDaveHello) [inline ajax/libs/fingerprintjs/package.json]: This won't match `fingerprint.js` in the latest version of fingerprintjs.

**Comment by pvnr0082t** [comment]
@kennynaoh I have updated the autoupdate config. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think it should add ```filename``` in ```package.json```
Is it correct?

**Comment by PeterDaveHello** [comment]
Wrong. See:
https://github.com/pvnr0082t/cdnjs/blob/add-autoupdate-fingerprintjs/ajax/libs/fingerprintjs/package.json#L3

**Comment by kennynaoh** [comment]
@PeterDaveHello 
Sorry about that.
I think the other things are correct.
Please help me review it again.
Thanks.

**Comment by PeterDaveHello** [comment]
Need a rebase upon latest master branch.

**Comment by pvnr0082t** [comment]
@kennynaoh I have done the rebase. Please help me review it again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think it has rebased to the latest master branch
And the other things are correct, too.
Please help me review it again.
Thanks.

**Pull request merged**